src/components/PhoneInput/index.tsx

```diff
@@ -1,46 +1,37 @@
-import { useEffect, useState } from 'react';
-import IntlTelInput from 'react-intl-tel-input';
+import { useState } from "react";
+import PhoneInputInternational from "react-phone-input-international";
+import "react-phone-input-international/lib/style.css";
```

Comment:
this import has to be moved to `src/index.css`, a fix for Next.js since we can't import CSS files in components like this

Reply:
I fixed

Reply:
> this import has to be moved to `src/index.css`, a fix for Next.js since we can't import CSS files in components like this

ok thank you,